packages/components/src/button/stories/index.story.tsx

```diff
@@ -44,52 +44,52 @@ const meta: Meta< typeof Button > = {
 };
 export default meta;
 
-const Template: Story< typeof Button > = ( props ) => {
+const Template: StoryFn< typeof Button > = ( props ) => {
 	return <Button { ...props }></Button>;
 };
 
-export const Default: Story< typeof Button > = Template.bind( {} );
+export const Default = Template.bind( {} );
```

Comment:
I'm curious why some stories have the type removed (`Story< typeof Button >` in this case) whereas in others, it is changed from `Story` to `StoryFn`, like in `ColorIndicator`: 

https://github.com/WordPress/gutenberg/blob/40e8b64828a0e29d1e93e40b52963ae3debbf005/packages/components/src/color-indicator/stories/index.story.tsx#L32

Reply:
Good question. These explicit typings weren't necessary to begin with, because `Template` is already typed.